podaac/subsetter/subset.py

```diff
@@ -1135,6 +1135,37 @@ def decode_cf_datetime(num_dates, units, calendar=None, use_cftime=None):
     xarray.coding.times.decode_cf_datetime = decode_cf_datetime
 
 
+def test_access_sst_dtime_values(datafile):
+    """
+    Test accessing values of 'sst_dtime' variable in a NetCDF file.
+
+    Parameters
+    ----------
+    datafile (str): Path to the NetCDF file.
+
+    Returns
+    -------
+    access_successful (bool): True if 'sst_dtime' values are accessible, False otherwise.
+    """
+
+    nc_dataset, _, _ = open_as_nc_dataset(datafile)
+    args = {
+        'decode_coords': False,
+        'mask_and_scale': True,
+        'decode_times': True
+    }
+    try:
+        with xr.open_dataset(
+                xr.backends.NetCDF4DataStore(nc_dataset),
+                **args
+        ) as dataset:
+            # pylint: disable=pointless-statement
+            dataset['sst_dtime'].values
```

Comment:
Rather than hardcoding a variable name here, let's loop through all dataset vars and test they are accessible.